modules/imgcodecs/src/loadsave.cpp

```diff
@@ -639,6 +749,90 @@ bool imreadmulti(const String& filename, std::vector<Mat>& mats, int flags)
     return imreadmulti_(filename, flags, mats);
 }
 
+
+/**
+* Read parts of a multi-page image
+*
+*  This function merely calls the actual implementation above and returns itself.
+*
+* @param[in] filename File to load
+* @param[in] mats Reference to C++ vector<Mat> object to hold the images
+* @param[in] start Start index for the sequence to load
+* @param[in] count number of images to load
+* @param[in] flags Flags you wish to set.
+*
+*/
```

Comment:
BTW, Doxygen comments are not used in .cpp files. They can be safely omitted.

Reply:
Thanks for the feedback